packages/ts-migrate-plugins/src/index.ts

```diff
@@ -16,7 +16,7 @@ import updateSourceText, {
   SourceTextUpdate as SourceTextUpdateType,
 } from './utils/updateSourceText';
 
-export type Plugin<T = {}> = PluginType<T>;
+export type Plugin<T = unknown> = PluginType<T>;
```

Comment:
Changes like this are to address [`@typescript-eslint/ban-types`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md).